articles/site-recovery/migrate-overview.md

```diff
@@ -34,7 +36,20 @@ Site Recovery should be used:
 - For disaster recovery of on-premises machines to Azure.
 - For disaster recovery of Azure VMs, between Azure regions.
 
-Although we recommend using Azure Migrate to migrate on-premises servers to Azure, if you've already started your migration journey with Site Recovery, you can continue using it to complete your migration.  
+## What Should I Use?
+
+We recommend using Azure Migrate to migrate on-premises servers to Azure.  However, if you've already started your migration journey with Site Recovery, consider the following:
+
+- With ASR already replicating your servers, there is no need to deploy a Migrate appliance, remove the BCDR protection, and replicate with a new appliance.
+- However, there are benefits to doing assessment, dependency analysis, and business case review with the Azure Migrate discovery appliance even for workloads that are already replicating.
+- There may be architecture changes needed to support the workload long term; in that case, address those but continue to use ASR to replicate so that you do not remove protections.
+
+In sum:
+
+- If you are beginning a new migration with neither ASR nor Migrate in place, use Azure Migrate.
+- If you have ASR in place actively protecting machines, continue to use it.
```

Comment:
Can we adjust this to "continue to use it for replication, but consider using Azure Migrate for business cases and dependency analysis" or something like that?

Reply:
@phorn30 ^ in case the above didn't notify you.